upload/src/addons/SV/StandardLib/_output/templates/admin/svStandardLib_template_view_modifications.html

```diff
@@ -6,33 +6,73 @@
 <xf:breadcrumb href="{{ link('templates/edit', $template, {'type': $template.type, 'style_id': $style.style_id}) }}">{$template.title}</xf:breadcrumb>
 
 <xf:css src="public:diff.less" />
+<xf:macro template="public:prism_macros" name="setup" />
 
-<div class="block">
+<xf:form action="{{ link('templates/view-modifications', $template, {'type': $template.type, 'style_id': $style.style_id}) }}"
+		 class="block"
+		 ajax="{$_xfWithData}" >
 	<div class="block-container">
-		<div class="block-body block-row block-body--contained">
-			<xf:form action="{{ link('templates/view-modifications', $template, {'type': $template.type, 'style_id': $style.style_id}) }}"
-					 class="block"
-					 ajax="{{ $_xfWithData ? 'true' : '' }}" >
-				<xf:checkboxrow rowtype="fullWidth noLabel">
-					<xf:foreach loop="$mods" value="$mod">
-						<xf:option name="id[]"
-								   value="{$mod.modification_id}"
-								   checked="{{ $activeMods.{$mod.modification_id} ? 'checked' : '' }}"
-								   label="{{ $mod.addon_id ? $mod.addon_id . ' - ' : '' }}{{ $mod.modification_key }} - {{ $mod.description }} (<a href='{{ link('template-modifications/edit', $mod) }}'>Edit</a>)" >
-							<xf:hint>{$status.{$mod->modification_id}}</xf:hint>
-						</xf:option>
-					</xf:foreach>
-				</xf:checkboxrow>
-				<xf:hiddenval name="reload">1</xf:hiddenval>
-				<xf:submitrow rowtype="simple" submit="Reload" />
-			</xf:form>
-			
-			<ol class="diffList diffList--code">
-				<xf:foreach loop="$diffs" value="$diff">
-					<xf:set var="$diffHtml">{{ $diff.1|join('<br />') }}</xf:set>
-					<li class="diffList-line diffList-line--{$diff.0}">{{ trim($diffHtml) !== '' ? $diffHtml : '&nbsp;' }}</li>
+		<div class="block-body">
+			<xf:checkboxrow rowtype="fullWidth noLabel">
+				<xf:foreach loop="$mods" value="$mod">
+					<xf:option name="id[]"
+							   value="{$mod.modification_id}"
+							   checked="{{ $activeMods.{$mod.modification_id} ? 'checked' : '' }}"
+							   label="{{ $mod.addon_id ? $mod.addon_id . ' - ' : '' }}{{ $mod.modification_key }} - {{ $mod.description }} (<a href='{{ link('template-modifications/edit', $mod) }}'>Edit</a>)" >
+						<xf:hint>{$status.{$mod->modification_id}}</xf:hint>
+					</xf:option>
 				</xf:foreach>
-			</ol>
+			</xf:checkboxrow>
 		</div>
+		
+		<xf:hiddenval name="reload">1</xf:hiddenval>
```

Comment:
This doesn't preserve the tab selection, meaning seeing the php code generated by whatever selection doesn't work.

Adding a simple `<xf:hiddenval name="tab">{$tab}</xf:hiddenval>`, and it reloads the entire tabs block.

Reply:
How easy would it be to just reload the tabs on submit rather than the entire page?